Resources/public/js/adapter/fancytree.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * A tree browser adapter for the Fancytree library.
+ *
+ * @author Wouter J <wouter@wouterj.nl>
+ * @see https://github.com/mar10/fancytree
+ */
+var FancytreeAdapter = function (data_url) {
+    if (!window.jQuery || !jQuery.fn.fancytree) {
+        throw 'The FancytreeAdapter requires both jQuery and the FancyTree library.';
+    }
+
+    var requestNodeToFancytreeNode = function (requestNode) {
+        var fancytreeNode = {
+            // fixme: remove ugly replacing by just not putting stuff like this in the JSON response
+            title: requestNode.data.replace(' (not editable)', ''),
```

Comment:
shouldn't we just adapt the tree adapter class for this? ah, that is in the phpcr-odm admin bundle right? there are other indicators for a 2.3 version of that anyways, in which we could bump the tree bundle requirement to 2.0